mountpoint-s3/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Other changes
 
 * Fix an issue where `fstat` would fail and return `ESTALE` when invoked on a file descriptor after a successful `fsync`. ([#1085](https://github.com/awslabs/mountpoint-s3/pull/1085))
+* Implement statfs to report non-zero value for total blocks, free blocks, free inodes and maximum file name length (before 0 was reported ([#1118](https://github.com/awslabs/mountpoint-s3/pull/1118))).
```

Comment:
I'd recommend something more like this, which can help the reader understand why this is important.

```suggestion
* Implement statfs to report non-zero synthetic values. This may unblock applications which rely on verifying there is available space before creating new files. ([#1118](https://github.com/awslabs/mountpoint-s3/pull/1118)))
```

I'd recommend moving the line above the previous line, since I think its more impactful for customers.

Reply:
Thanks, that reads much clearer. Fixed this.